From bc9e80720ac06ca2c8c940713151f1fe111ae32b Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Wed, 13 Jul 2016 09:47:22 +0800
Subject: [PATCH 1203/5242] MLK-12928-22: mfd: pf1550: correct VMINSYS_MASK
 define

commit  39b3482d271bca0771525c56435c9e7bd7b8d389 from
https://source.codeaurora.org/external/imx/linux-imx.git

correct VMINSYS_MASK and _CNFG_REGTEMP_MASK define for regmap

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/mfd/pf1550.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/mfd/pf1550.h b/include/linux/mfd/pf1550.h
index 7e7a7d9..7d0a13e 100644
--- a/include/linux/mfd/pf1550.h
+++ b/include/linux/mfd/pf1550.h
@@ -156,9 +156,9 @@ enum pf1550_pmic_reg {
 
 #define PF1550_CHARG_REG_BATT_REG_CHGCV_MASK		0x3f
 #define PF1550_CHARG_REG_BATT_REG_VMINSYS_SHIFT		6
-#define PF1550_CHARG_REG_BATT_REG_VMINSYS_MASK		0x3
+#define PF1550_CHARG_REG_BATT_REG_VMINSYS_MASK		(0x3 << 6)
 #define PF1550_CHARG_REG_THM_REG_CNFG_REGTEMP_SHIFT	2
-#define PF1550_CHARG_REG_THM_REG_CNFG_REGTEMP_MASK	0x3
+#define PF1550_CHARG_REG_THM_REG_CNFG_REGTEMP_MASK	(0x3 << 2)
 
 #define PMIC_IRQ_SW1_LS		BIT(0)
 #define PMIC_IRQ_SW2_LS		BIT(1)
-- 
1.7.9.5

